parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -334,6 +369,22 @@ public static Float16LogicalTypeAnnotation float16Type() {
     return Float16LogicalTypeAnnotation.INSTANCE;
   }
 
+  public static GeometryLogicalTypeAnnotation geometryType(String crs) {
+    return new GeometryLogicalTypeAnnotation(crs);
+  }
+
+  public static GeometryLogicalTypeAnnotation geometryType() {
+    return new GeometryLogicalTypeAnnotation("OGC:CRS84");
+  }
```

Comment:
```suggestion
  public static GeometryLogicalTypeAnnotation geometryType() {
    return new GeometryLogicalTypeAnnotation(null);
  }
```

Setting the default to null may save some bytes in the thrift message because we don't need to serialize crs. We have to write the string if the default is `"OGC:CRS84"`.

Reply:
Maybe we don't have to provide `geometryType()` at all?